src/core/document.js

```diff
@@ -14,11 +14,10 @@
  */
 
 import { Catalog, ObjectLoader, XRef } from './obj';
+import { createPromiseCapability, error, info, isArrayBuffer, isNum,
```

Comment:
Format this in the same way as before, so `import {` on one line and the actual imports start on the next line. Finally, `} from ...` should also be on a new line.

Reply:
Got it, thank you. I wasn't sure for such formatting rules and relied upon lint validation